### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by ccobasi**
- Refactor calculator class to function

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import calculate from './logic/calculator';
+
+function Calculator() {
```
Comment (bellom) [inline src/components/Calculator.js]: Good job jobs with Hooks, but we need to make use of ES6 i.e Arrow function.

Example:

```
const Calculator = () => {
......
}
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    if (currCalc.total === null) {
+      currCalc.total = '0';
+    }
+    if (currCalc.total !== null) {
+      display.innerText = currCalc.next;
+    } else {
+      display.innerText = currCalc.total;
+    }
+    calcUpdate(currCalc);
+  };
```
Comment (bellom) [inline src/components/Calculator.js]: Let's fix the `updateCalc`, `result`, and `operators` seem not to show.

KIndly make sure the calculator is working correctly.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ccobasi  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well using Hooks, setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi @ccobasi 👋 , 
## STATUS: APPROVED 👍 
Great job implementing changes requested by the previous reviewer. 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**